sdcm/cluster.py

```diff
@@ -1265,15 +1271,53 @@ def filter_backtraces(backtrace):
                     if backtrace['event'].raw_backtrace:
                         try:
                             scylla_debug_info = self.get_scylla_debuginfo_file()
-                            output = self.remoter.run(
-                                'addr2line -Cpife {0} {1}'.format(scylla_debug_info, " ".join(backtrace['event'].raw_backtrace.split('\n'))), verbose=False)
+                            LOGGER.error("scylla-debug-file: %s", scylla_debug_info)
+                            # output = self.remoter.run(
+                            #     'addr2line -Cpife {0} {1}'.format(scylla_debug_info, " ".join(backtrace['event'].raw_backtrace.split('\n'))), verbose=False)
+                            output = self.decode_backtrace(scylla_debug_info, " ".join(
```

Comment:
@fruch  Thanks direction. Thought about queue for monitoring node for decoding. But will it correctly have same datetime and correctly displayed via annotations if it will be decoded consistently via queue?

Reply:
timestamp are from the creation of the event.
and in the DbLogEvent is should be extracted from the log line itself.

so queues should be affecting that, it would be visible after we decode, but the timestamp would should be correct.

Reply:
also antheor optimization can be caching identical traceback, so we won't decode the same one again and again, if it happened lots of times on multiple nodes, we'll have an already decode version of it.

Reply:
Adding queue and thread.